.github/workflows/run_tests.yml

```diff
@@ -24,7 +24,7 @@ jobs:
 
   test_cover:
     name: Coverage
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-24.04
```

Comment:
can be just latest?

Reply:
It was a fixed version, it is a fixed version.

Reply:
yes, so this commit might not even exist